includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -696,6 +699,11 @@ private function getCommunity() {
 		return $data;
 	}
 
+	private function isLocalSitemapAvailable() {
+		$default = true; // $wgEnableLocalSitemapPageExt = true; in CommonSettings
```

Comment:
The idea is take the value from CommonSettings and then override from WikiFactory for given wiki. The value of $wgEnableLocalSitemapPageExt would be the value for specifically www wiki, not the one from CommonSettings :-( . That's why I needed to hardcode it. We had a discussion about it on #wikia-design-system